src/AppServices/KafkaPage/KafkaPage.tsx

```diff
@@ -35,50 +49,212 @@ export const KafkaPage: FunctionComponent = () => {
+                    <DescriptionListGroup>
+                      <DescriptionListTerm>
+                        {t("dataTransfer")}
+                      </DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$0.09/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                    <DescriptionListGroup>
+                      <DescriptionListTerm>{t("storage")}</DescriptionListTerm>
+                      <DescriptionListDescription>
+                        {"$0.10/GB"}
+                      </DescriptionListDescription>
+                    </DescriptionListGroup>
+                  </DescriptionList>
+                </FlexItem>
+              </Flex>
+              <Divider isVertical />
+              <FlexItem flex={{ default: "flex_1" }}>
+                <TextContent>
+                  <Text component={TextVariants.p}>
+                    <Trans
+                      i18nKey={"kafkaoverview:streamingUnitText"}
+                      values={{
+                        value: "streaming unit",
+                      }}
+                    />
+                  </Text>
+                </TextContent>
+              </FlexItem>
+            </Flex>
+          </CardBody>
+        </Card>
+      </MarketingPageSection>
+      <MarketingPageSection kafkaOverviewPage={true}>
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h2">{t("kafkaInstanceCapacityTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Stack hasGutter>
+              <Text component={TextVariants.p}>
+                {t("kafkaInstanceCapacityDescription")}
+              </Text>
+              <KafkaInstanceCapacityTable />
+            </Stack>
+          </CardBody>
+          <CardFooter>
+            <TextContent className={"pf-u-font-size-sm"}>
+              {t("kafkaInstanceCapacityfooter1")}{" "}
+              <Button
+                data-testid="redhatSupport-buttonCTA"
+                isInline
+                variant={ButtonVariant.link}
+                component="a"
+                target="_blank"
+                href="https://access.redhat.com/support"
+              >
+                {" "}
+                Redhat support
+                <ExternalLinkAltIcon className="pf-u-ml-xs" />
+              </Button>{" "}
+              {t("kafkaInstanceCapacityfooter2")}
+              <Button
+                data-testid="diskSpaceMetrics-buttonCTA"
+                isInline
+                variant={ButtonVariant.link}
+                component="a"
+                target="_blank"
+                href="https://access.redhat.com/documentation/en-us/red_hat_openshift_streams_for_apache_kafka/1/guide/aced8e5e-8229-4cb2-82f9-87a8caa24bb3"
+              >
+                {" "}
+                monitoring disk space metrics
+                <ExternalLinkAltIcon className="pf-u-ml-xs" />
```

Comment:
What's odd about the inline link with icon is that the icon will wrap to the next line and display by itself. We can keep the text from breaking before the icon by adding a non-breaking space `&nbsp;`.


![image](https://user-images.githubusercontent.com/21063328/165848698-c8d6ef83-478a-4fe0-a85c-739e59ab442a.png)